opamp/observiq/observiq_downloadable_file_manager.go

```diff
@@ -24,14 +27,15 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
+	"strings"
 
-	archiver "github.com/mholt/archiver/v3"
 	"github.com/observiq/bindplane-agent/opamp"
 	"github.com/open-telemetry/opamp-go/protobufs"
 	"go.uber.org/zap"
 )
 
 const extractFolder = "latest"
+const maxArchiveObjectByteSize = 1024
```

Comment:
Using this in conjunction with `io.CopyN` to resolve a gosec warning for a decompression bomb. This is the max size in bytes we will copy from a source file. Arbitrarily chose 1024, not sure what we want to do for this?

Reply:
An object is a file, right? One of those files is going to be the collector, much bigger than this (if I'm understanding correctly). Let's set this to like 1GB.

Reply:
Yea no that's a good point, 1GB it is.